studio.html

```diff
@@ -50,7 +50,7 @@
                 .querySelector('meta[name="aem-base-url"]')
                 ?.setAttribute('content', baseUrl);
         </script>
-        <script src="studio/src/maslib.js"></script>
+        <script src="web-components/dist/mas.js" type="module"></script>
```

Comment:
it is too early to switch studio to use mas web-components.
lets do this change after we switch milo or around that time. we still have a lot of work to do before we are ready for the switch